src/Services/QueuedJobService.php

```diff
@@ -1533,6 +1579,271 @@ private function addJobHandlersToLogger(LoggerInterface $logger, QueuedJob $job,
         }
     }
 
+    /**
+     * Find any eligible jobs that qualify for an automated job retry and schedule their retry
+     *
+     * @throws ValidationException
+     */
+    protected function retryEligibleJobs(int $queueType): void
```

Comment:
```suggestion
    protected function markEligibleJobsForRetry(int $queueType): void
```
We're not actually _performing_ the retry here - we're just marking these jobs to be retried next time the queue runs.

Could use the word "schedule" instead of "mark" if you like, as that's the word used in the PHPDoc comment.